mobile/src/main/res/values/strings.xml

```diff
@@ -46,6 +46,12 @@
     <string name="settings_openhab_icon_format_value_svg" translatable="false">SVG</string>
     <string name="settings_openhab_icon_format_png">Bitmap</string>
     <string name="settings_openhab_icon_format_svg">Vector</string>
+    <string name="settings_chart_scaling">Chart scaling</string>
+    <string name="settings_chart_scaling_summary">Note: Smaller scaling shrinks fonts and line widths, so there\'s more space for the graph, but font readability may suffer.</string>
```

Comment:
I've seen that Android itself also uses abbreviation like `there's`, so it should be changed in openHAB to be consistently. WDYT @maniac103 @digitaldan?